package.json

```diff
@@ -51,12 +51,13 @@
     "karma-mocha": "~2.0",
     "karma-sourcemap-loader": "~0.3",
     "karma-spec-reporter": "0.0.32",
-    "karma-webpack": "~4.0",
-    "mocha": "~9",
+    "karma-webpack": "^5.0.0",
+    "mocha": "^10.1.0",
     "sass": "~1",
-    "sass-loader": "~10",
+    "sass-loader": "^13.2.0",
     "should": "~13.2",
     "sinon": "~9",
+    "util": "^0.12.5",
     "vue-cli-plugin-i18n": "~2"
```

Comment:
We are using latest version of this, and it has moderate level vulnerability. 

This is a PR to fix this but the maintainer is MIA since July: https://github.com/intlify/vue-cli-plugin-i18n/pull/279